components/clients/styles.js

```diff
@@ -0,0 +1,75 @@
+import { StyleSheet } from "react-native";
```

Comment:
I think it makes sense to keep style files related to the file. So if there is a ContactIcon.js file, we'd have a ComtactIcon.styles.js and so on